nixos/modules/services/networking/ndppd.nix

```diff
@@ -91,7 +91,7 @@ let
           auto: Same as iface, but instead of manually specifying the outgoing
             interface, check for a matching route in /proc/net/ipv6_route.
         '';
-        default = "auto";
+        default = "static";
```

Comment:
Why change the default?

Reply:
I mentioned it in the commit message, didn't you see it?
https://github.com/NixOS/nixpkgs/pull/352258/commits/47a329f255e60725bbd86451b9415e802e6114cd
https://manpages.opensuse.org/Tumbleweed/ndppd/ndppd.conf.5.en.html#static
Instead I'm asking why the default was set to "auto"?

Reply:
Hmm. I don't know. That default was set in #55122.

Reply:
Then set it to "static" to make things right.